### ISSUE TITLE
How can I help list?

### ISSUE THREAD

**Issue by ccoffrin** [csplib/csplib#111]
Collect a list of small outstanding tasks and curate those on the website, to help inspire would-be volunteers.

For example, the outstanding items from the Essence Catalog, and how to do a migration.

**Comment by turingfan** [comment]
This is a great idea... I assume you mean the main website, i.e. not the github?

**Comment by ccoffrin** [comment]
Yup!

**Comment by 9thbit** [comment]
This could just be a matter of adding them as issues on the repo's issue trackers and tagging them 'minor' or something. It is what it is for after all. :)

**Comment by turingfan** [comment]
That's true but I think it would be nice to have a more general page on csplib.org with maybe some simpler way to interact than by pull request.

**Comment by ccoffrin** [comment]
A mix of both could work.  Bullet points on the home page that link to
detailed discriptions in GitHub issues or other similar documentation pages.
